docs/user/_index.md

```diff
@@ -93,41 +94,9 @@ developer team.
 
 ## Traces
 
-Grafana Agent supports collecting traces and sending them to Tempo using its
+Grafana Agent collects traces and forwards them to Tempo using its
 `traces` subsystem. This is done using the upstream [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector).
 Agent can ingest OpenTelemetry, OpenCensus, Jaeger, Zipkin, or Kafka spans.
-See documentation on how to configure [receivers]({{< relref "./configuration/traces-config.md" >}}).
+For more information on how to configure, see [receivers]({{< relref "./configuration/traces-config.md" >}}).
```

Comment:
'refer to' instead of 'see'. Also, is `receivers` the name of the actual page that the user goes to? The link name should be the same name as the H1 or whatever subsection you are sending them to. Capitalization applies.